app/models/reservation.rb

```diff
@@ -1,4 +1,3 @@
 class Reservation < ApplicationRecord
-  belongs_to :user
-  belongs_to :car
+  validates :user_id, :car_id, :city, :date, length: { maximum: 20 }
 end
```

Comment:
Technically this preserves the relation between tables user-reservations-cars using the Ids (car_id, user_id) but why you deleted the belongs_to ? does it give you trouble?

Reply:
Based on how the user authentication was made, I implemented it this way, there is already a method to determine the current user on redux, as well as the chosen car, I would need to change a lot of things in order to make this work the relation work.